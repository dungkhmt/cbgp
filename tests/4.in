200 200
200 526
72 65
121 36
12 5
43 1
178 28
87 42
135 13
143 1
154 127
88 173
129 13
42 185
10 5
6 123
11 113
142 113
13 114
63 78
146 60
25 60
13 49
26 154
21 95
112 4
85 120
78 21
12 160
14 1
76 142
46 91
13 198
80 60
38 44
27 156
76 113
29 28
3 9
3 27
194 40
59 193
1 166
120 103
17 51
7 30
41 30
182 197
173 96
8 141
82 164
114 126
73 11
79 54
44 1
59 186
151 39
143 169
24 164
113 174
103 25
53 108
83 12
92 65
172 65
17 93
50 85
27 64
37 22
118 85
12 165
22 196
71 22
63 68
6 1
25 137
1 123
140 24
7 12
112 182
102 99
167 88
10 85
9 36
138 13
55 88
129 39
129 165
104 28
41 25
129 140
132 46
54 161
41 42
72 100
107 159
149 197
126 168
27 116
1 3
173 200
21 88
9 69
73 14
4 7
114 190
172 34
88 63
95 88
182 149
60 79
109 64
87 180
180 60
10 51
50 10
53 124
48 39
173 22
133 82
46 75
136 11
135 12
5 144
25 139
12 4
109 27
180 146
197 176
8 149
3 21
153 123
31 6
98 39
129 151
20 113
161 87
73 191
25 54
15 143
162 1
36 1
40 48
99 26
42 25
22 4
59 50
143 147
31 64
53 49
78 2
173 177
31 44
98 48
115 73
134 173
1 18
8 22
60 161
119 107
72 47
39 199
130 7
155 73
99 25
20 11
65 181
64 1
129 198
60 61
199 133
46 25
76 174
11 123
40 13
17 67
61 79
2 43
1 38
14 106
26 12
22 138
10 144
24 22
46 103
14 143
1 21
198 135
3 109
134 4
193 188
25 107
153 69
3 64
107 7
110 101
137 12
4 144
32 46
195 88
138 24
4 176
25 12
184 54
96 84
90 104
36 57
74 22
32 10
118 186
12 56
26 110
121 9
30 42
70 57
146 42
26 34
1 11
189 44
60 87
85 91
122 107
31 189
9 123
13 24
103 159
175 1
119 103
52 134
102 45
53 48
130 193
2 21
49 194
167 195
99 101
123 166
94 13
112 8
6 64
21 36
8 200
23 12
150 57
12 139
114 16
22 140
69 11
11 179
56 129
123 69
156 3
59 85
168 16
53 89
9 27
100 65
51 81
73 143
51 75
29 104
86 79
140 164
172 127
37 13
53 13
192 15
17 172
12 30
101 26
95 167
11 35
71 24
31 175
40 49
177 196
52 8
182 4
189 38
88 117
53 40
120 119
84 190
135 56
65 127
75 111
172 100
64 44
32 85
92 81
173 187
7 25
64 66
171 85
86 60
178 104
199 82
140 71
18 175
118 59
24 104
24 152
11 153
4 1
161 86
137 30
108 40
13 12
8 134
25 26
118 120
163 13
127 26
152 28
96 37
97 65
136 179
103 107
174 142
96 88
114 36
18 31
162 14
68 88
114 4
4 8
58 114
143 19
74 138
4 140
96 187
32 91
59 130
125 77
200 22
34 10
127 34
114 163
96 145
171 103
33 20
160 4
48 13
9 6
155 11
175 38
59 119
26 65
92 17
97 26
74 24
93 92
39 13
9 106
149 4
17 72
1 2
21 195
84 37
160 140
12 99
14 15
120 59
158 35
69 170
75 10
159 25
60 42
29 90
122 159
145 21
148 19
95 117
80 7
28 71
146 185
183 28
152 183
53 39
5 13
5 4
182 176
184 87
78 55
67 72
6 166
21 55
80 42
66 6
53 105
134 141
65 93
84 58
99 110
65 17
28 22
129 164
172 128
60 185
128 100
138 37
84 114
10 17
117 167
80 25
59 188
157 139
10 12
67 47
103 132
98 94
8 173
132 91
151 133
82 13
139 99
21 43
114 1
129 82
187 37
113 69
7 144
37 114
20 142
98 13
171 120
168 114
83 10
1 136
36 190
7 23
63 4
129 135
31 38
183 24
194 124
30 80
168 13
46 51
145 84
42 184
156 109
86 61
45 26
177 22
133 129
37 173
129 160
188 130
19 15
25 157
57 62
59 7
17 47
31 66
58 37
85 103
84 36
130 10
90 178
172 72
65 47
181 172
148 169
34 12
176 149
13 4
23 4
53 194
18 6
56 165
45 99
196 173
113 170
13 16
22 29
37 177
39 94
25 184
92 46
4 88
34 83
17 34
4 141
46 10
181 100
188 50
39 82
124 89
69 76
194 89
25 79
82 24
110 102
28 90
25 30
12 157
128 72
28 24
109 116
24 29
9 21
107 59
57 121
52 173
78 88
194 105
73 148
129 94
110 131
149 141
97 25
179 1
2 4
136 155
108 48
173 4
2 63
129 12
26 77
71 152
34 154
88 145
163 16
105 89
10 7
97 92
50 130
1
365 2 208 100 158 154 297 9 77 11 169 167 211 13 185 85 0 0 80 14 188 172 146 81 30 10 384 0 374 5 27 12 119 50 174 161 373 6 183 177 158 153 170 84 212 14 136 115 169 23 136 36 162 159 156 86 163 87 186 38 173 160 105 28 194 188 120 52 187 170 13 7 146 135 173 157 123 100 127 92 186 37 187 36 209 101 170 156 144 44 110 34 122 44 131 91 118 93 101 19 107 37 193 93 123 99 183 30 206 100 139 83 11 4 145 136 104 16 197 25 183 27 10 2 208 98 166 155 115 37 172 161 123 45 208 97 174 174 14 1 151 85 125 43 200 184 137 116 107 36 180 180 141 40 207 99 181 26 199 24 108 38 127 105 119 53 147 141 108 27 183 28 186 30 194 96 123 96 161 88 106 29 111 35 114 38 133 89 202 104 150 138 135 35 132 90 151 37 124 41 146 38 147 40 165 20 160 90 124 95 10 0 174 17 128 93 162 156 146 39 107 35 191 86 177 176 17 8 377 6 163 157 202 103 109 21 112 18 111 19 8 1 169 22 170 169 122 97 143 40 26 14 131 38 125 42 141 88 98 15 145 40 107 30 127 102 194 92 138 82 193 174 185 39 137 117 152 38 149 83 194 91 182 178 367 4 210 12 148 142 188 32 367 3 372 7 194 90 12 2 126 101 153 87 174 173 132 39 192 175 161 157 153 39 189 170 168 21 148 82 184 29 363 1 27 11 143 89 140 84 170 168 204 102 27 13 370 7 177 175 113 21 130 39 192 95 180 179 175 158 195 88 183 97 160 89 193 173 187 31 124 40 192 87 154 88 184 32 189 33 109 20 149 136 101 18 172 158 16 9 196 188 375 6 101 17 119 94 205 101 183 96 194 89 137 83 126 103 184 86 
