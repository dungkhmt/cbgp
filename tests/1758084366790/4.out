(71, 64), (120, 35), (11, 4), (42, 0), (177, 27), (86, 41), (134, 12), (142, 0), (153, 126), (87, 172), (128, 12), (41, 184), (9, 4), (5, 122), (10, 112), (141, 112), (12, 113), (62, 77), (145, 59), (24, 59), (12, 48), (25, 153), (20, 94), (111, 3), (84, 119), (77, 20), (11, 159), (13, 0), (75, 141), (45, 90), (12, 197), (79, 59), (37, 43), (26, 155), (75, 112), (28, 27), (2, 8), (2, 26), (193, 39), (58, 192), (0, 165), (119, 102), (16, 50), (6, 29), (40, 29), (181, 196), (172, 95), (7, 140), (81, 163), (113, 125), (72, 10), (78, 53), (43, 0), (58, 185), (150, 38), (142, 168), (23, 163), (112, 173), (102, 24), (52, 107), (82, 11), (91, 64), (171, 64), (16, 92), (49, 84), (26, 63), (36, 21), (117, 84), (11, 164), (21, 195), (70, 21), (62, 67), (5, 0), (24, 136), (0, 122), (139, 23), (6, 11), (111, 181), (101, 98), (166, 87), (9, 84), (8, 35), (137, 12), (54, 87), (128, 38), (128, 164), (103, 27), (40, 24), (128, 139), (131, 45), (53, 160), (40, 41), (71, 99), (106, 158), (148, 196), (125, 167), (26, 115), (0, 2), (172, 199), (20, 87), (8, 68), (72, 13), (3, 6), (113, 189), (171, 33), (87, 62), (94, 87), (181, 148), (59, 78), (108, 63), (86, 179), (179, 59), (9, 50), (49, 9), (52, 123), (47, 38), (172, 21), (132, 81), (45, 74), (135, 10), (134, 11), (4, 143), (24, 138), (11, 3), (108, 26), (179, 145), (196, 175), (7, 148), (2, 20), (152, 122), (30, 5), (97, 38), (128, 150), (19, 112), (160, 86), (72, 190), (24, 53), (14, 142), (161, 0), (35, 0), (39, 47), (98, 25), (41, 24), (21, 3), (58, 49), (142, 146), (30, 63), (52, 48), (77, 1), (172, 176), (30, 43), (97, 47), (114, 72), (133, 172), (0, 17), (7, 21), (59, 160), (118, 106), (71, 46), (38, 198), (129, 6), (154, 72), (98, 24), (19, 10), (64, 180), (63, 0), (128, 197), (59, 60), (198, 132), (45, 24), (75, 173), (10, 122), (39, 12), (16, 66), (60, 78), (1, 42), (0, 37), (13, 105), (25, 11), (21, 137), (9, 143), (23, 21), (45, 102), (13, 142), (0, 20), (197, 134), (2, 108), (133, 3), (192, 187), (24, 106), (152, 68), (2, 63), (106, 6), (109, 100), (136, 11), (3, 143), (31, 45), (194, 87), (137, 23), (3, 175), (24, 11), (183, 53), (95, 83), (89, 103), (35, 56), (73, 21), (31, 9), (117, 185), (11, 55), (25, 109), (120, 8), (29, 41), (69, 56), (145, 41), (25, 33), (0, 10), (188, 43), (59, 86), (84, 90), (121, 106), (30, 188), (8, 122), (12, 23), (102, 158), (174, 0), (118, 102), (51, 133), (101, 44), (52, 47), (129, 192), (1, 20), (48, 193), (166, 194), (98, 100), (122, 165), (93, 12), (111, 7), (5, 63), (20, 35), (7, 199), (22, 11), (149, 56), (11, 138), (113, 15), (21, 139), (68, 10), (10, 178), (55, 128), (122, 68), (155, 2), (58, 84), (167, 15), (52, 88), (8, 26), (99, 64), (50, 80), (72, 142), (50, 74), (28, 103), (85, 78), (139, 163), (171, 126), (36, 12), (52, 12), (191, 14), (16, 171), (11, 29), (100, 25), (94, 166), (10, 34), (70, 23), (30, 174), (39, 48), (176, 195), (51, 7), (181, 3), (188, 37), (87, 116), (52, 39), (119, 118), (83, 189), (134, 55), (64, 126), (74, 110), (171, 99), (63, 43), (31, 84), (91, 80), (172, 186), (6, 24), (63, 65), (170, 84), (85, 59), (177, 103), (198, 81), (139, 70), (17, 174), (117, 58), (23, 103), (23, 151), (10, 152), (3, 0), (160, 85), (136, 29), (107, 39), (12, 11), (7, 133), (24, 25), (117, 119), (162, 12), (126, 25), (151, 27), (95, 36), (96, 64), (135, 178), (102, 106), (173, 141), (95, 87), (113, 35), (17, 30), (161, 13), (67, 87), (113, 3), (3, 7), (57, 113), (142, 18), (73, 137), (3, 139), (95, 186), (31, 90), (58, 129), (124, 76), (199, 21), (33, 9), (126, 33), (113, 162), (95, 144), (170, 102), (32, 19), (159, 3), (47, 12), (8, 5), (154, 10), (174, 37), (58, 118), (25, 64), (91, 16), (96, 25), (73, 23), (92, 91), (38, 12), (8, 105), (148, 3), (16, 71), (0, 1), (20, 194), (83, 36), (159, 139), (11, 98), (13, 14), (119, 58), (157, 34), (68, 169), (74, 9), (158, 24), (59, 41), (28, 89), (121, 158), (144, 20), (147, 18), (94, 116), (79, 6), (27, 70), (145, 184), (182, 27), (151, 182), (52, 38), (4, 12), (4, 3), (181, 175), (183, 86), (77, 54), (66, 71), (5, 165), (20, 54), (79, 41), (65, 5), (52, 104), (133, 140), (64, 92), (83, 57), (98, 109), (64, 16), (27, 21), (128, 163), (171, 127), (59, 184), (127, 99), (137, 36), (83, 113), (9, 16), (116, 166), (79, 24), (58, 187), (156, 138), (9, 11), (66, 46), (102, 131), (97, 93), (7, 172), (131, 90), (150, 132), (81, 12), (138, 98), (20, 42), (113, 0), (128, 81), (186, 36), (112, 68), (6, 143), (36, 113), (19, 141), (97, 12), (170, 119), (167, 113), (82, 9), (0, 135), (35, 189), (6, 22), (62, 3), (128, 134), (30, 37), (182, 23), (193, 123), (29, 79), (167, 12), (45, 50), (144, 83), (41, 183), (155, 108), (85, 60), (44, 25), (176, 21), (132, 128), (36, 172), (128, 159), (187, 129), (18, 14), (24, 156), (56, 61), (58, 6), (16, 46), (30, 65), (57, 36), (84, 102), (83, 35), (129, 9), (89, 177), (171, 71), (64, 46), (180, 171), (147, 168), (33, 11), (175, 148), (12, 3), (22, 3), (52, 193), (17, 5), (55, 164), (44, 98), (195, 172), (112, 169), (12, 15), (21, 28), (36, 176), (38, 93), (24, 183), (91, 45), (3, 87), (33, 82), (16, 33), (3, 140), (45, 9), (180, 99), (187, 49), (38, 81), (123, 88), (68, 75), (193, 88), (24, 78), (81, 23), (109, 101), (27, 89), (24, 29), (11, 156), (127, 71), (27, 23), (108, 115), (23, 28), (8, 20), (106, 58), (56, 120), (51, 172), (77, 87), (193, 104), (72, 147), (128, 93), (109, 130), (148, 140), (96, 24), (178, 0), (1, 3), (135, 154), (107, 47), (172, 3), (1, 62), (128, 11), (25, 76), (70, 151), (33, 153), (87, 144), (162, 15), (104, 88), (9, 6), (96, 91), (49, 129), 
numberIntersection=28929 
minAngle=0.0 
sumAngle=38.881512098875824 
minEdgeLength=6.0 
sumEdgeLength=139898.2155166622 
minDistance=0.0 
sumDistance=2.2806459408027884E7 

-28929.0000000000 0.0000000000 38.8815120989 6.0000000000 139898.2155166622 0.0000000000 22806459.4080278840
0: (300, 400),
1: (300, 406),
2: (300, 413),
3: (200, 400),
4: (300, 420),
5: (300, 427),
6: (300, 434),
7: (300, 441),
8: (400, 400),
9: (300, 449),
10: (300, 457),
11: (300, 466),
12: (300, 476),
13: (300, 486),
14: (400, 412),
15: (300, 498),
16: (400, 425),
17: (300, 512),
18: (400, 437),
19: (400, 450),
20: (300, 528),
21: (300, 546),
22: (300, 569),
23: (300, 597),
24: (286, 600),
25: (271, 600),
26: (400, 464),
27: (256, 600),
28: (243, 600),
29: (229, 600),
30: (400, 477),
31: (400, 492),
32: (400, 507),
33: (216, 600),
34: (400, 524),
35: (203, 600),
36: (190, 600),
37: (177, 600),
38: (164, 600),
39: (150, 600),
40: (400, 542),
41: (400, 561),
42: (136, 600),
43: (122, 600),
44: (400, 582),
45: (400, 606),
46: (400, 633),
47: (107, 600),
48: (100, 582),
49: (400, 665),
50: (400, 702),
51: (100, 557),
52: (100, 537),
53: (400, 746),
54: (100, 520),
55: (100, 505),
56: (399, 800),
57: (100, 492),
58: (100, 481),
59: (369, 800),
60: (341, 800),
61: (314, 800),
62: (100, 471),
63: (100, 461),
64: (288, 800),
65: (262, 800),
66: (237, 800),
67: (100, 453),
68: (212, 800),
69: (188, 800),
70: (100, 445),
71: (163, 800),
72: (138, 800),
73: (100, 437),
74: (112, 800),
75: (86, 800),
76: (59, 800),
77: (100, 430),
78: (31, 800),
79: (100, 423),
80: (1, 800),
81: (100, 416),
82: (100, 410),
83: (100, 403),
84: (0, 746),
85: (0, 702),
86: (0, 665),
87: (100, 397),
88: (0, 633),
89: (0, 606),
90: (0, 582),
91: (0, 561),
92: (0, 542),
93: (100, 390),
94: (100, 384),
95: (100, 377),
96: (0, 524),
97: (100, 370),
98: (100, 363),
99: (0, 507),
100: (0, 492),
101: (0, 477),
102: (0, 464),
103: (0, 450),
104: (0, 437),
105: (0, 425),
106: (100, 355),
107: (0, 412),
108: (0, 400),
109: (0, 388),
110: (0, 375),
111: (100, 347),
112: (0, 363),
113: (100, 339),
114: (0, 350),
115: (0, 336),
116: (100, 329),
117: (0, 323),
118: (0, 308),
119: (0, 293),
120: (0, 276),
121: (0, 258),
122: (100, 319),
123: (0, 239),
124: (0, 218),
125: (100, 308),
126: (0, 194),
127: (0, 167),
128: (100, 295),
129: (100, 280),
130: (0, 135),
131: (0, 98),
132: (0, 54),
133: (100, 263),
134: (100, 243),
135: (100, 218),
136: (107, 200),
137: (122, 200),
138: (136, 200),
139: (150, 200),
140: (164, 200),
141: (1, 0),
142: (177, 200),
143: (190, 200),
144: (203, 200),
145: (31, 0),
146: (59, 0),
147: (86, 0),
148: (216, 200),
149: (112, 0),
150: (138, 0),
151: (163, 0),
152: (188, 0),
153: (212, 0),
154: (237, 0),
155: (262, 0),
156: (229, 200),
157: (288, 0),
158: (314, 0),
159: (243, 200),
160: (341, 0),
161: (256, 200),
162: (271, 200),
163: (286, 200),
164: (300, 203),
165: (300, 231),
166: (300, 254),
167: (300, 272),
168: (369, 0),
169: (399, 0),
170: (400, 54),
171: (400, 98),
172: (300, 288),
173: (400, 135),
174: (300, 302),
175: (300, 314),
176: (300, 324),
177: (400, 167),
178: (300, 334),
179: (400, 194),
180: (400, 218),
181: (300, 343),
182: (400, 239),
183: (400, 258),
184: (400, 276),
185: (400, 293),
186: (300, 351),
187: (400, 308),
188: (400, 323),
189: (300, 359),
190: (400, 336),
191: (400, 350),
192: (400, 363),
193: (400, 375),
194: (300, 366),
195: (300, 373),
196: (300, 380),
197: (300, 387),
198: (400, 388),
199: (300, 394),
300 400 300 406 300 413 200 400 300 420 300 427 300 434 300 441 400 400 300 449 300 457 300 466 300 476 300 486 400 412 300 498 400 425 300 512 400 437 400 450 300 528 300 546 300 569 300 597 286 600 271 600 400 464 256 600 243 600 229 600 400 477 400 492 400 507 216 600 400 524 203 600 190 600 177 600 164 600 150 600 400 542 400 561 136 600 122 600 400 582 400 606 400 633 107 600 100 582 400 665 400 702 100 557 100 537 400 746 100 520 100 505 399 800 100 492 100 481 369 800 341 800 314 800 100 471 100 461 288 800 262 800 237 800 100 453 212 800 188 800 100 445 163 800 138 800 100 437 112 800 86 800 59 800 100 430 31 800 100 423 1 800 100 416 100 410 100 403 0 746 0 702 0 665 100 397 0 633 0 606 0 582 0 561 0 542 100 390 100 384 100 377 0 524 100 370 100 363 0 507 0 492 0 477 0 464 0 450 0 437 0 425 100 355 0 412 0 400 0 388 0 375 100 347 0 363 100 339 0 350 0 336 100 329 0 323 0 308 0 293 0 276 0 258 100 319 0 239 0 218 100 308 0 194 0 167 100 295 100 280 0 135 0 98 0 54 100 263 100 243 100 218 107 200 122 200 136 200 150 200 164 200 1 0 177 200 190 200 203 200 31 0 59 0 86 0 216 200 112 0 138 0 163 0 188 0 212 0 237 0 262 0 229 200 288 0 314 0 243 200 341 0 256 200 271 200 286 200 300 203 300 231 300 254 300 272 369 0 399 0 400 54 400 98 300 288 400 135 300 302 300 314 300 324 400 167 300 334 400 194 400 218 300 343 400 239 400 258 400 276 400 293 300 351 400 308 400 323 300 359 400 336 400 350 400 363 400 375 300 366 300 373 300 380 300 387 400 388 300 394 0: (300, 400), 1: (300, 406), 2: (300, 413), 3: (200, 400), 4: (300, 420), 5: (300, 427), 6: (300, 434), 7: (300, 441), 8: (400, 400), 9: (300, 449), 10: (300, 457), 11: (300, 466), 12: (300, 476), 13: (300, 486), 14: (400, 412), 15: (300, 498), 16: (400, 425), 17: (300, 512), 18: (400, 437), 19: (400, 450), 20: (300, 528), 21: (300, 546), 22: (300, 569), 23: (300, 597), 24: (286, 600), 25: (271, 600), 26: (400, 464), 27: (256, 600), 28: (243, 600), 29: (229, 600), 30: (400, 477), 31: (400, 492), 32: (400, 507), 33: (216, 600), 34: (400, 524), 35: (203, 600), 36: (190, 600), 37: (177, 600), 38: (164, 600), 39: (150, 600), 40: (400, 542), 41: (400, 561), 42: (136, 600), 43: (122, 600), 44: (400, 582), 45: (400, 606), 46: (400, 633), 47: (107, 600), 48: (100, 582), 49: (400, 665), 50: (400, 702), 51: (100, 557), 52: (100, 537), 53: (400, 746), 54: (100, 520), 55: (100, 505), 56: (399, 800), 57: (100, 492), 58: (100, 481), 59: (369, 800), 60: (341, 800), 61: (314, 800), 62: (100, 471), 63: (100, 461), 64: (288, 800), 65: (262, 800), 66: (237, 800), 67: (100, 453), 68: (212, 800), 69: (188, 800), 70: (100, 445), 71: (163, 800), 72: (138, 800), 73: (100, 437), 74: (112, 800), 75: (86, 800), 76: (59, 800), 77: (100, 430), 78: (31, 800), 79: (100, 423), 80: (1, 800), 81: (100, 416), 82: (100, 410), 83: (100, 403), 84: (0, 746), 85: (0, 702), 86: (0, 665), 87: (100, 397), 88: (0, 633), 89: (0, 606), 90: (0, 582), 91: (0, 561), 92: (0, 542), 93: (100, 390), 94: (100, 384), 95: (100, 377), 96: (0, 524), 97: (100, 370), 98: (100, 363), 99: (0, 507), 100: (0, 492), 101: (0, 477), 102: (0, 464), 103: (0, 450), 104: (0, 437), 105: (0, 425), 106: (100, 355), 107: (0, 412), 108: (0, 400), 109: (0, 388), 110: (0, 375), 111: (100, 347), 112: (0, 363), 113: (100, 339), 114: (0, 350), 115: (0, 336), 116: (100, 329), 117: (0, 323), 118: (0, 308), 119: (0, 293), 120: (0, 276), 121: (0, 258), 122: (100, 319), 123: (0, 239), 124: (0, 218), 125: (100, 308), 126: (0, 194), 127: (0, 167), 128: (100, 295), 129: (100, 280), 130: (0, 135), 131: (0, 98), 132: (0, 54), 133: (100, 263), 134: (100, 243), 135: (100, 218), 136: (107, 200), 137: (122, 200), 138: (136, 200), 139: (150, 200), 140: (164, 200), 141: (1, 0), 142: (177, 200), 143: (190, 200), 144: (203, 200), 145: (31, 0), 146: (59, 0), 147: (86, 0), 148: (216, 200), 149: (112, 0), 150: (138, 0), 151: (163, 0), 152: (188, 0), 153: (212, 0), 154: (237, 0), 155: (262, 0), 156: (229, 200), 157: (288, 0), 158: (314, 0), 159: (243, 200), 160: (341, 0), 161: (256, 200), 162: (271, 200), 163: (286, 200), 164: (300, 203), 165: (300, 231), 166: (300, 254), 167: (300, 272), 168: (369, 0), 169: (399, 0), 170: (400, 54), 171: (400, 98), 172: (300, 288), 173: (400, 135), 174: (300, 302), 175: (300, 314), 176: (300, 324), 177: (400, 167), 178: (300, 334), 179: (400, 194), 180: (400, 218), 181: (300, 343), 182: (400, 239), 183: (400, 258), 184: (400, 276), 185: (400, 293), 186: (300, 351), 187: (400, 308), 188: (400, 323), 189: (300, 359), 190: (400, 336), 191: (400, 350), 192: (400, 363), 193: (400, 375), 194: (300, 366), 195: (300, 373), 196: (300, 380), 197: (300, 387), 198: (400, 388), 199: (300, 394), 
Time taken: 0.159 s
